packages/perseus-score/src/widgets/free-response/score-free-response.ts

```diff
@@ -0,0 +1,23 @@
+import type {
+    PerseusFreeResponseUserInput,
+    PerseusFreeResponseRubric,
+    PerseusScore,
+} from "@khanacademy/perseus-core";
+
+function scoreFreeResponse(
+    userInput: PerseusFreeResponseUserInput,
+    rubric: PerseusFreeResponseRubric,
+    locale: string,
+): PerseusScore {
+    // TODO: Implement support for external scoring. Since the user's input is
+    //       free text input, we can't easily do scoring inside the widget, so
+    //       we'll need to have a way to do it by some other method. For now,
+    //       we'll just always give full credit.
+    return {
+        type: "points",
+        earned: 1,
+        total: 1,
+    };
```

Comment:
Do you mind using `scoreNoop`? It can do the same thing, it's just easier for us to find when we need to look into widgets that aren't actually doing real scoring. I know this is temporary, but just in case.